features/checkout/skipping_payment_step_when_order_total_is_zero_after_applying_coupon.feature

```diff
@@ -0,0 +1,27 @@
+@checkout
+Feature: Skipping payment selection when order total is zero after applying coupon
+    In order not to select payment method when it is unnecessary
+    As a Customer
+    I want to be redirected directly to order summary page after shipping selection
```

Comment:
`I want to be redirected directly to order summary page after shipping selection when my order total is zero` ?